docs/DEVELOPMENT_GUIDE.md

```diff
@@ -0,0 +1,111 @@
+# Development Guide
+
+## How to Contribute
+
+We'd love to accept your patches and contributions to this project. There are
```

Comment:
I think it's ok to link to CONTRIBUTING.md here instead of copying that content here.

Reply:
Done.